src/lib/bootloader/grub2_widgets.rb

```diff
@@ -535,9 +522,9 @@ def store
         console_value = Yast::UI.QueryWidget(Id(:console_args), :Value)
         BootloaderFactory.current.enable_serial_console(console_value)
       elsif use_gfxterm
-        grub_default.terminal = :gfxterm
+        grub_default.terminal = [:gfxterm]
       else
-        grub_default.terminal = :console
+        grub_default.terminal = [:console]
```

Comment:
so maybe current logic with gfxterm versus console is correct. But user should be able to set in parallel serial console and console/gfxterm.